cli/pg_datanymizer/src/app.rs

```diff
@@ -50,7 +52,11 @@ impl App {
                 self.options.pg_dump_args.clone(),
             )?
             .dump(&mut connection),
-        }
+        }?;
+
+        self.print_new_version_if_available();
```

Comment:
Maybe on this flag we will not dump data, just check the version?

Reply:
Done.